### PR TITLE
1ES migration

### PR DESCRIPTION
**Pull request by danieljurek**
Pipeline run: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3640617&view=logs&j=19992227-62fb-5b50-4e29-3b72bd33eea1&t=19992227-62fb-5b50-4e29-3b72bd33eea1

There are a couple warnings that are classed as errors but do not block the pipeline at this time.

### CODE REVIEW

**Comment by timotheeguerin** [comment]
hhm we count on the publishing to be done automatically when version are bumped, this is a significant change in the flow.

**Review by scbedd — APPROVED** [review]
LGTM!

**Review by scbedd — APPROVED** [review]

File: eng/pipelines/templates/stages/1es-redirect.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      sourceAnalysisPool:
+        name: azsdk-pool-mms-win-2022-general
+        image: azsdk-pool-mms-win-2022-1espt
+        os: windows
+      psscriptanalyzer:
+        compiled: true
+        break: true
+      eslint:
+        enabled: false
+        justificationForDisabling: "ESLint injected task has failures because it uses an old version of mkdirp. We should not fail for tools not controlled by the repo. See: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3556850"
```
Comment (timotheeguerin) [inline eng/pipelines/templates/stages/1es-redirect.yml:35]: So this will always show up as a warning?

**Review by timotheeguerin — APPROVED** [review]

File: eng/pipelines/templates/stages/1es-redirect.yml (hunk 1 above)
Comment (scbedd) [reply eng/pipelines/templates/stages/1es-redirect.yml:35]: For now, yes. We are in a desperate rush to finish up the last of the pipeline migration before 3/31.

After that (or before that, depending on how these last few go), we will get the 1ES ESLint team to fix their step.

**Pull request merged**